src/main/java/space/obminyashka/items_exchange/api/ApiKey.java

```diff
@@ -31,6 +31,8 @@ public class ApiKey {
     public static final String CATEGORY_ALL = CATEGORY + "/all";
     public static final String CATEGORY_ID = CATEGORY + "/{category_id}";
     public static final String CATEGORY_SIZES = CATEGORY_ID + "/sizes";
+    // Mail API
+    public static final String MAIL_VALIDATE_TOKEN = API + "/email/validate/{token_id}";
```

Comment:
I think you should separate this String "/email/validate/{token_id}" by new variable